### PR TITLE
[TASK][PATCH] Add MIT and Apache2 License

### PR DESCRIPTION
**Pull request by tuyennn**
### Description
Add MIT and Apache2 License

+ This will add MIT and Apache2 license for user to choose

### CODE REVIEW

File: mage2gen/license.py [hunk 1]
```diff
@@ -71,3 +71,13 @@ class OSLV3(FileLicense):
...
 	identifier = 'OSL-3.0'
+
+class MIT(FileLicense):
+	template_license = 'mit.txt'
+	template_short_license = 'mit.txt'
+	identifier = 'MIT'
+
+class APACHE2(FileLicense):
+	template_license = 'apache2.txt'
+	template_short_license = 'apache2_short.txt'
```
Comment (krukas) [inline mage2gen/license.py]: Apache also has a short license (https://www.apache.org/licenses/LICENSE-2.0#apply)

Comment (tuyennn) [reply mage2gen/license.py]: Apache2 Short Version already added in commit.

Comment (krukas) [reply mage2gen/license.py]: lol somehow overlooked that :)

**Comment by dheesbeen** [comment]
@tuyennn Can you also create a pull request magento-2.1 branch and the magento-2.3 branch? Thanks a lot

**Comment by tuyennn** [comment]
@dheesbeen I saw it was already on magento-2.1 and magento-2.3 branch

**Comment by dheesbeen** [comment]
@tuyennn Sorry! @lewisvoncken has merged it into 2.1 and 2.3.

**Comment by lewisvoncken** [comment]
@tuyennn sorry for the delay but this has just gone live. There also needed to be made some changes to the implementation in the frontend.

**Comment by tuyennn** [comment]
@lewisvoncken  @dheesbeen No problem at all, well we have time but there are other things like family and works. Well done for the 2.3 Model support

**Pull request merged**